pkg/clients/generated/apis/compute/v1beta1/computeforwardingrule_types.go

```diff
@@ -36,12 +36,10 @@ import (
 )
 
 type ForwardingruleFilterLabels struct {
-	/* Immutable. Name of the metadata label. The length must be between
-	1 and 1024 characters, inclusive. */
+	/* Immutable. Name of the metadata label. The length must be between 1 and 1024 characters, inclusive. */
```

Comment:
I'm unsure why the format changed in generated type...🥲 I do not think that will break anything, do we want to fix this?